src/sig_stateful/xmss/sig_stfl_xmss_sha256.c

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: MIT
+
+#include <string.h>
+#include <stdlib.h>
+
+#include <oqs/oqs.h>
+#include "./external/xmss.h"
+#include "./external/params.h"
+#include "sig_stfl_xmss_xmssmt.h"
+#include "./external/xmss_namespace.h"
+
+// ======================== XMSS10-SHA256 ======================== //
+
+OQS_API OQS_STATUS OQS_SIG_STFL_alg_xmss_sha256_h10_keypair(uint8_t *public_key, OQS_SECRET_KEY *secret_key) {
+	if (secret_key == NULL || public_key == NULL) {
+		return OQS_ERROR;
+	}
+
+	uint32_t oid = 0x00000001;
+	if (xmss_keypair(public_key, secret_key, oid) != 0) {
+		return OQS_ERROR;
+	}
+	return OQS_SUCCESS;
+}
+
+OQS_SIG_STFL *OQS_SIG_STFL_alg_xmss_sha256_h10_new(void) {
+
+	OQS_SIG_STFL *sig = (OQS_SIG_STFL *)malloc(sizeof(OQS_SIG_STFL));
+	if (sig == NULL) {
+		return NULL;
+	}
+	memset(sig, 0, sizeof(OQS_SIG_STFL));
+
+	sig->method_name = "XMSS-SHA2_10_256";
+	sig->alg_version = "...";
+
+	// Check how true this is
+	sig->claimed_nist_level = 2;
```

Comment:
Need to decide whether we're keeping the claimed_nist_level or not.

Reply:
I have removed it from implementation